### DISCUSSION TITLE
Minecraft Newbie Q: can I create my own users? Are there open-source clients?

### DISCUSSION THREAD

**Discussion by nodje** [itzg/docker-minecraft-server#1160] [Q&A]
That may sound stupid, but I'm totally new to Minecraft.

There doesn't seem to be any explanation on user creation and management, only that by default you identify with a minecraft.net account, if I'm not mistaken.

Aren't there any way to create your own user to play on a server?
I mean, once you have the server, why bother using the official Minecraft.net ?

Then how do you get a free client? 
Is using something like this online JS client https://github.com/michaljaz/webmc possible?

**Comment by itzg** [comment]
You buy a copy and you get to create a user then:

https://www.minecraft.net/en-us/store/minecraft-java-edition/
